dags/openshift_nightlies/tasks/benchmarks/e2e.py

```diff
@@ -29,8 +29,24 @@ def __init__(self, dag, config: DagConfig, release: OpenshiftRelease, task_group
         self.es_server_baseline = var_loader.get_secret("es_server_baseline")
 
         # Specific Task Configuration
-        self.vars = var_loader.build_task_vars(
-            release=self.release, task=self.task_group)
+
```

Comment:
this seems very cumbersome to manage especially if we add new config_files for different tasks. It'd be more maintainable if we just expanded the manifest to let each profile set it's config_file

Reply:
ahh makes sense